css/comp-page.css

```diff
@@ -129,8 +129,9 @@
   line-height: 1.5em;
   color: #464d5f;
```

Comment:
i thnk you forget to use color variable here

Reply:
ack, done